packages/sitecore-jss/src/site/graphql-error-pages-service.ts

```diff
@@ -0,0 +1,100 @@
+import { GraphQLClient, GraphQLRequestClient } from '../graphql';
+import { siteNameError } from '../constants';
+import debug from '../debug';
+
+// The default query for request error handling
+const defaultQuery = /* GraphQL */ `
+  query ErrorPagesQuery($siteName: String!, $language: String!) {
+    site {
+      siteInfo(site: $siteName) {
+        errorHandling(language: $language) {
+          notFoundPagePath
+          serverErrorPagePath
+        }
+      }
+    }
+  }
+`;
+
+export type GraphQLErrorPagesServiceConfig = {
+  /**
+   * Your Graphql endpoint
+   */
+  endpoint: string;
+  /**
+   * The API key to use for authentication
+   */
+  apiKey: string;
+  /**
+   * The JSS application name
+   */
+  siteName: string;
+  /**
+   * The language
+   */
+  language: string;
+};
+
+export type ErrorPagesType = { notFoundPagePath: string; serverErrorPagePath: string };
```

Comment:
Seems we can drop "Type" (redundant)?
```suggestion
export type ErrorPages = { notFoundPagePath: string; serverErrorPagePath: string };
```